java/org/apache/catalina/ha/tcp/SimpleTcpCluster.java

```diff
@@ -339,6 +339,13 @@ public void setChannelSendOptions(int channelSendOptions) {
         this.channelSendOptions = channelSendOptions;
     }
 
+    public void setChannelSendOptions(String channelSendOptions) {
+
+        int value = Channel.parseChannelSendOptions(channelSendOptions);
+        if (value > 0)
```

Comment:
Hi,

From code format point of view: We use curly braces even when there is only one statement in "if", "for" blocks.

Regards,
Violeta